crates/ide-completion/src/render/function.rs

```diff
@@ -77,8 +80,9 @@ fn render(
         .as_assoc_item(ctx.db())
         .and_then(|trait_| trait_.containing_trait_or_trait_impl(ctx.db()))
         .map_or(false, |trait_| completion.is_ops_trait(trait_));
+
     item.set_relevance(CompletionRelevance {
-        type_match: compute_type_match(completion, &ret_type),
```

Comment:
Looks like functions were only being type matched based on the return type. I am not sure if there was a reason for this :thinking: